packages/protocol/package.json

```diff
@@ -15,7 +15,7 @@
     "test:coverage": "pnpm test && pnpm coverage",
     "generate:genesis": "ts-node ./utils/generate_genesis/main.ts",
     "test:genesis": "./test/genesis/generate_genesis.test.sh",
-    "test:integration": "./test/test_integration.sh",
+    "test:integration": "K_BLOCK_MAX_GAS_LIMIT=30000000 K_COMMIT_DELAY_CONFIRMS=1 ./test/test_integration.sh",
```

Comment:
and these pre-processor env vars also will be removed after tokenomics get merged? ref: https://github.com/taikoxyz/taiko-mono/pull/478

Reply:
Yep I can update this now that the tokenomics PR is merged in to use the LibConfig instead.

Reply:
Just re-approved the tokenomics PR, then seems it got merged automatically 👍